benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -102,7 +107,16 @@ private BenchmarkExecutionResult executeBenchmark(Benchmark benchmark, Optional<
         BenchmarkExecutionResultBuilder resultBuilder = new BenchmarkExecutionResultBuilder(benchmark);
         List<QueryExecutionResult> executions;
         try {
-            executeQueries(benchmark, benchmark.getPrewarmRuns(), false, executionTimeLimit);
+            executions = executeQueries(benchmark, benchmark.getPrewarmRuns(), true, executionTimeLimit);
+            executions.stream()
+                    .filter(execution -> execution.getFailureCause() != null &&
+                            execution.getFailureCause().getClass().equals(QueryExecutionDriver.ResultComparisonException.class))
+                    .findFirst()
```

Comment:
It's enough to find one such error; also for standard benchmarks each benchmark contains only single query.

Reply:
Standard = not throughput. In benchto nomenclature each query is a separate benchmark (for standard benchmarks). So this loop will iterate over different runs of the same query.

Reply:
Interesting, that I did not know. Even if it is normally used like that I guess it could be used with different queries, right? IF that's the case I would still add all errors, maybe grouping by query to de-duplicate different runs (I should not be needed because you run comparison just for the first run anyway)